site/docs/components/overview.mdx

```diff
@@ -27,6 +27,23 @@ We strive to achieve the following quality attributes on all of our components:
 
 The status of these can be found on the documentation page for each component, or on the [Component health](/component-health) page, which shows the status of all components.
 
+### API
+- **Avoid language overloading**: Avoid prop names that are used in HTML, such as `type`.
+- **Lean on existing mental models**: Things like input `onChange` should not stray from the default behavior that React has defined.
+- **Encapsulation:** Components should only be concerned about their own form and function and not about how they appear in relation to other components.
```

Comment:
I'm curious if this is enough information for non-design system folk. Should this include an example? The best one I can think of is no padding / margin around the component.